core/src/main/java/org/radargun/config/StageHelper.java

```diff
@@ -54,6 +55,15 @@ private static void addStage(String stageName, Class<? extends org.radargun.Stag
       }
    }
 
+   private static String handleLegacyNamespace(String namespace) {
```

Comment:
Uh, that's a nasty hack. What about adding a `deprecatedName` to `@Namespace` and properly registering the stage in `StageHelper.addStage`?